### ISSUE TITLE
FXIOS-777 ⁃ [XCUITests] Download testing page is down causing Download tests failures

### ISSUE THREAD

**Issue by isabelrios** [mozilla-mobile/firefox-ios#7130]
There is testing site we use to download different files used for the Download tests it is down:
http://demo.borland.com/testsite/download_testpage.php

Logs with tests failures.
https://dashboard.buddybuild.com/apps/57bf25c0f096bc01001e21e0/build/5f3490917297df00010a049a#tests

There is a new site we could use but we will need to modify the tests.
https://www.thinkbroadband.com/download

If the problem is not solved shortly, tests will be updated

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-777)

**Comment by isabelrios** [comment]
I have been investigating to try to use a custom test site for Downloads to be added to our test-fixtures pages but even though I get the download prompt on Desktop, I can't get it on iOS using same page. Something I don't understand may be managed in a different way on mobile. 

I used, among other simplier options, the script I found [here](http://danml.com/download.html). But either I get the downloaded file in a different tab or the download does not start, the Download prompt on iOS is not shown.

While this is investigated it may be good to adpat these tests to use the other testing site or disabled them.
